gradoop-examples/src/main/java/org/gradoop/utils/converter/deprecated/sources/OldCSVDataSource.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.utils.converter.deprecated.sources;
+
+
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.tuple.Tuple3;
+import org.gradoop.common.model.impl.pojo.Edge;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.flink.io.api.DataSource;
+import org.gradoop.flink.io.impl.csv.CSVBase;
+import org.gradoop.flink.io.impl.csv.metadata.MetaData;
+import org.gradoop.flink.model.api.epgm.GraphCollection;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.util.GradoopFlinkConfig;
+
+/**
+ * A graph data source for CSV files.
+ *
+ * The datasource expects files separated by vertices and edges, e.g. in the following directory
+ * structure:
+ *
+ * csvRoot
```

Comment:
Put those lines in  a `<pre><code> ... </code></pre>` block to avoid unwanted newlines in Javadoc.